src/message/messageActionSheet.js

```diff
@@ -199,6 +201,9 @@ export const constructMessageActionButtons = ({
     buttons.push('addReaction');
   }
   if (!isAnOutboxMessage(message) && !isTopicNarrow(narrow) && !isPrivateNarrow(narrow)) {
+    if (isAnnouncementOnly) {
+      reply.title = 'Navigate to topic';
+    }
```

Comment:
I feel that `Navigate to topic` is much better than `Reply` for this action anyway, because it **does not** open up a reply dialog as expected, and actually performs the action of navigating to topic.

This has often been confusing to me, and to other users as well, I think.

Reply:
Will it be alright if I completely replace the `Reply` button with `Navigate to topic` button?